docs/examples/cart-service/README.md

```diff
@@ -75,7 +75,7 @@ export class CartChanged {
     readonly quantity: number
   ) {}
 
-  public entityID(): string {
+  public entityId(): string {
```

Comment:
This won't compile if you use the camelcase version, we're using all-caps for IDs and acronyms as a convention